rocAL/rocAL/source/api/rocal_api_meta_data.cpp

```diff
@@ -438,7 +497,7 @@ void ROCAL_API_CALL rocalBoxEncoder(RocalContext p_context, std::vector<float>&
 }
 
 void
-ROCAL_API_CALL rocalCopyEncodedBoxesAndLables(RocalContext p_context, float* boxes_buf, int* labels_buf)
+ROCAL_API_CALL rocalCopyEncodedBoxesAndLables(RocalContext p_context, double* boxes_buf, int* labels_buf)
```

Comment:
Have we made sure we tested the box encoder SSD example with the double here ?

Reply:
I have not tested any other coco pipeline which might be affected by this change. Since the new SSD pipeline also requires double bboxes, I figure we'll making the other changes in another PR. Will be discussing this with shobana and decide on the next course of action

Reply:
Added the bbox dtype as float64 in COCO pipeline example which returns non-zero bboxes. When I checked the dumped images, the bboxes were printed correctly